code/modules/sanity/breakdown.dm

```diff
@@ -51,9 +52,17 @@
 	return TRUE
 
 /datum/breakdown/proc/conclude()
+	holder.insight += 10
```

Comment:
Swap it for changeLevel(10) or any good number, because /datum/sanity/proc/updateLevel(new_level) will update for you the insight later down the code.

Reply:
I need direct insight insertion, changelevel would affect sanity and I don't need it.